composer.json

```diff
@@ -17,6 +17,11 @@
             "ZendService\\Amazon\\": "library/"
         }
     },
+    "autoload-dev": {
+        "psr-0": {
+            "ZendServiceTest\\Amazon\\": "tests/ZendServiceTest/Amazon/"
```

Comment:
It's correct. I've tried each, and tried different sets of paths. The `autoload` entry is *verbatim* from the previous, and looks exactly like this as well.

We cannot do `"ZendServiceTest\\": "tests/ZendServiceTest/"`, as we want to be more specific than that; thus the `Amazon` segments. I originally assumed PSR-4 would work as well, but had autoloading issues when I changed the autoloading type and ran `dump-autoload`. Reverting and re-dumping the autoloader caused it to work.